Sources/MapboxCoreNavigation/BundleAdditions.swift

```diff
@@ -27,18 +27,33 @@ extension Bundle {
      The Mapbox Core Navigation framework bundle.
      */
     public class var mapboxCoreNavigation: Bundle {
-        return Bundle(for: RouteController.self)
+        get {
+            #if SWIFT_PACKAGE
+            return .module
+            #else
+            return Bundle(for: RouteController.self)
+            #endif
+        }
     }
     
     /**
      The Mapbox Navigation framework bundle, if installed.
      */
     class var mapboxNavigationIfInstalled: Bundle? {
-        // Assumption: MapboxNavigation.framework includes NavigationViewController and exposes it to the Objective-C runtime as MapboxNavigation.NavigationViewController.
-        guard let NavigationViewController = NSClassFromString("MapboxNavigation.NavigationViewController") else {
+        get {
+            #if SWIFT_PACKAGE
+            for bundleIdentifier in Bundle.allBundles.compactMap({ $0.bundleIdentifier }) {
```

Comment:
👍 This is a much more elegant approach than the Objective-C runtime tricks I was thinking of. It happens to work when SPM is building static frameworks, because the bundle we’re looking for would be a resource bundle rather than a framework bundle.